src/main/java/org/opentripplanner/graph_builder/module/osm/WayPropertySet.java

```diff
@@ -452,17 +443,25 @@ private WayProperties applyMixins(
     double secondWalk = walkSafetyFeatures.back();
     for (WayProperties properties : mixins) {
       if (right) {
-        secondBicycle *= properties.getBicycleSafetyFeatures().back();
-        secondWalk *= properties.getWalkSafetyFeatures().back();
+        if (properties.getBicycleSafetyFeatures() != null) {
+          secondBicycle *= properties.getBicycleSafetyFeatures().back();
```

Comment:
Not your change but can you use "back"/"forward" instead of "first"/"second" in the variable name?